devtools/aptos-cargo-cli/src/lib.rs

```diff
@@ -198,6 +204,31 @@ impl AptosCargoCommand {
                 println!("Skipping targeted compiler v2 tests because no relevant files or packages were affected!");
                 Ok(())
             },
+            AptosCargoCommand::TargetedFrameworkUpgradeTests(_) => {
+                // Determine if the framework upgrade tests should be run.
+                // Start by calculating the changed files and affected packages.
+                let (_, _, changed_files) = package_args.identify_changed_files()?;
+                let (_, _, affected_package_paths) =
+                    self.get_args_and_affected_packages(package_args)?;
+
+                // Determine if any relevant files or packages were changed
+                let relevant_changes_detected = detect_relevant_changes(
+                    RELEVANT_FILE_PATHS_FOR_FRAMEWORK_UPGRADE_TESTS.to_vec(),
+                    RELEVANT_PACKAGES_FOR_FRAMEWORK_UPGRADE_TESTS.to_vec(),
+                    changed_files,
+                    affected_package_paths,
+                );
+
+                // Output if relevant changes were detected that require the framework upgrade
+                // test. This will be consumed by Github Actions and used to run the test.
+                // TODO: is there a cleaner way to output this for Github Actions?
```

Comment:
Yes?
```
export RESULT=$(cargo x targeted-framework-upgrade-tests | awk -F'Framework upgrade test required: ' '{print $2}')
```
this could be cleaner by not having an `awk` component. Just have the cargo tool plint what we actually want to go in `$RESULT`

Reply:
Hmm... I was thinking of something cleaner (i.e., not having to collect the printed string output).

Also, I think it's clearer if we print `Framework upgrade test required: true` instead of just `true`. In the latter case, it's hard to tell what `true` represents, e.g., does it mean the test was run and it passed? At least here, the full string makes it obvious.

Reply:
it gets processed through to `run_framework_upgrade_test=true` which gets interpreted by GHA. Should we skip directly to that string in Rust instead of doing string processing in bash-in-yaml?